src/__tests__/enhancer-test.js

```diff
@@ -7,41 +8,54 @@ import React, {Component} from 'react';
 
 describe('Enhancer', () => {
   it('sets up initial state', () => {
-    class Composed extends Component {}
+    class Composed extends Component {
+      render() {
+        return <div />;
+      }
+    }
     const Enhanced = Enhancer(Composed);
 
-    const instance = new Enhanced();
+    const ref = React.createRef();
+    renderFcIntoDocument(<Enhanced ref={ref} />);
```

Comment:
Class components are now wrapped in a HoC for injecting the contexts, so using `ref` to get access to the instance